fout = open('nfile')
print(fout.read())
